jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/kafka/SDXDepositorTopicsTest.java

```diff
@@ -0,0 +1,25 @@
+package us.dot.its.jpo.ode.kafka;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.ConfigDataApplicationContextInitializer;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(initializers = ConfigDataApplicationContextInitializer.class)
+@EnableConfigurationProperties(value = SDXDepositorTopics.class)
+class SDXDepositorTopicsTest {
+
+    @Autowired
+    SDXDepositorTopics sdxDepositorTopics;
+
+    @Test
+    void getInput() {
+        assertEquals("topic.SDWDepositorInput", sdxDepositorTopics.getInput());
+    }
```

Comment:
If the user modifies the configuration to use a different topic name, will this test have to be updated as well?

Reply:
Nope, the test [application.yaml](https://github.com/CDOT-CV/jpo-ode/blob/6607c1c7721e0bf861b003577fc2383e2bc1d4e5/jpo-ode-svcs/src/test/resources/application.yaml#L92) drives the values in this test and the other configuration tests. The only time this test will fail is if the test application yaml is adjusted or if there were changes to the SDXDepositorTopics class itself